### PR TITLE
Tests for the Tabs component

### PR DESCRIPTION
**Pull request by igorbrasileiro**
#### What is the purpose of this pull request?
as the title says

#### What problem is this solving?
Related issue #1083 

#### How should this be manually tested?
`yarn test`

#### Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/Tabs/index.test.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      const { getByTestId } = render(
+        <Tabs testId={tabsId}>
+          <Tab label="label 1" active>
+            Tab 1
+          </Tab>
+          <Tab label="label 2">Tab 2</Tab>
+        </Tabs>
+      )
+
+      expect(getByTestId(tabsId)).toHaveTextContent('Tab 1')
```
Comment (klzns) [inline react/components/Tabs/index.test.tsx]: [Avoid using getByTestId when possible](https://testing-library.com/docs/guide-which-query)

```suggestion
      expect(getByText('Tab 1')).toBeTruthy()
```

Comment (igorbrasileiro) [reply react/components/Tabs/index.test.tsx]: Nice @klzns, I will rethink about the way I did my tests.

Comment (emersonlaurentino) [reply react/components/Tabs/index.test.tsx]: but you don't need to remove the `testId` prop from your component, test only.

Comment (igorbrasileiro) [reply react/components/Tabs/index.test.tsx]: I removed from tab because if I want to get it, I can use getByText that's suggested by `Testing Library`

**Pull request merged**